http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/Boot.scala

```diff
@@ -209,6 +206,9 @@ object Boot extends IOApp {
         implicit0(ctx: Ask[IO, AppContext]) = Ask.const[IO, AppContext](
           AppContext(TraceId(s"Boot_${start}"), start)
         )
+//       TODO: this will be needed once we support more environments.
```

Comment:
I feel like this TODO should have a ticket attached to it in the comment so we can ctrl+f for it when the time comes

Reply:
I agree in general. The real issue for this is in fiabs, Sam is not guranteed to boot up before Leonardo is. I'm not entirely sure how we get around that yet. So punting on creating a concrete ticket...but this TODO will definitely be addressed at some point in Q2, so I'm not too concerned about having a ticket since we'll have to address this as we roll out to more envs.